omemo/litesessionstore.py

```diff
@@ -53,13 +56,13 @@ def getSubDeviceSessions(self, recipientId):
         deviceIds = [r[0] for r in result]
         return deviceIds
 
-    def getDeviceTuples(self):
-       q = "SELECT recipient_id, device_id from sessions"
-       c = self.dbConn.cursor()
-       result = []
-       for row in c.execute(q):
-           result.append((row[0],row[1]))
-       return result 
+    def getActiveDeviceTuples(self):
+        q = "SELECT recipient_id, device_id FROM sessions WHERE active = ?"
```

Comment:
Why don't just use `active = 1`? This way you don't need to have this weird `TRUE/FALSE` constants. Especially because you only fetch active device tupples in this function